irc/server.py

```diff
@@ -435,6 +435,25 @@ def handle_dump(self, params):
             for client in channel.clients:
                 print("     ", client.nick, client)
 
+    def handle_ison(self, params):
+        for nick in params.split(" "):
+            if len(params) == 0 or params.isspace():
+                response = '461: ISON Not enough parameters'
+                return response
+
+            first = True
```

Comment:
You can avoid using this flag `first` if you use a list and concatenate it at the end.
e. g.:
```
nicks = list()
for client in self.server.clients:
    if client == nick:
        nicks.append(nick)
response += response + ' '.join(nicks)
```

Reply:
Thank you - I will work to clean this whole commit up, thank you for being constructive. My first time trying to contribute to something like this and criticism like this is exactly what I'm looking for. (As well as to better the project!)